ui/fields/_db.php

```diff
@@ -8,7 +8,7 @@
 <input<?php PodsForm::attributes( $attributes, $name, $form_field_type, $options ); ?> />
 <script>
 	jQuery( function ( $ ) {
-		$( 'input#<?php echo esc_js( $attributes['id'] ); ?>' ).change( function () {
+		$( 'input#<?php echo esc_js( $attributes['id'] ); ?>' ).on( 'change', function () {
```

Comment:
:no_entry_sign: Barf out, gag me with a spoon! esc_js()!This is for inline javascript, which is against our standards. Use json_encode instead.

<sub>hash: ac1e84b6c5871e71fdb3b2ea351e870e</sub>